net/openssh/files/sshd.config

```diff
@@ -0,0 +1,6 @@
+config sshd
+	option enable '1'
```

Comment:
`enable` -> `enabled`

Reply:
It was enable in dropbear so I thought I should keep it enable here too.